src/main/java/seedu/duke/Util.java

```diff
@@ -0,0 +1,22 @@
+package seedu.duke;
+
+public class Util {
+    public static final int USER_JAVA_INDEX_DIFF = 1;
+    public static final String TASK_INDEX_KEY = "t";
+    public static final String TASK_NAME_KEY = "n";
+    public static final String DATE_KEY = "d";
+    public static final String PROJECT_INDEX_KEY = "p";
+    public static final int MINUTES_IN_HOUR_INT = 60;
+    public static final double MINUTES_IN_HOUR_DOUBLE = 60.0;
+    public static final String HOUR_INDEX_KEY = "h";
+    public static final String MINUTE_INDEX_KEY = "m";
+    public static final String MEMBER_INDEX_KEY = "m";
+    public static final String INDEX_NON_INTEGER = "indexNonInteger";
+    public static final String INVALID_TEAM_MEMBER_ID = "invalidTeamMemberID";
+    public static final String INVALID_PROJECT_ID = "invalidProjectID";
+    public static final String INVALID_MEMBER_ID = "invalidMemberID";
+    public static final String INVALID_TASK_ID = "invalidTaskID";
+    public static final String TASK_NOT_DONE = "taskNotDone";
+    public static final String WRONG_DATE_FORMAT = "WrongDateFormat";
+    public static final String EMPTY_PROJECT_LIST = "emptyProjectList";
```

Comment:
Should we change everything in the Exception class then??

Reply:
Yeah I think it'll be good!